src/stream.js

```diff
@@ -21,8 +26,63 @@ module.exports.handler = async (event) => {
 	delete(groupedRecords['deleteMe']); // delete bad events
 
 	for (const records of Object.entries(groupedRecords)) {
-		const latestRecord = records.pop();
+		logger.debug({records}, 'Records');
+		const latestRecord = records.pop().pop();
+		// Tag for a transient record that should be ignored.
+		if (latestRecord.date === '_current') {
+			continue;
+		}
 		logger.debug({latestRecord}, 'Event received');
-	}
 
+		const responses = latestRecord.users && Object.keys(latestRecord.users).map(key => {
+			return {
+				name: `<@${key}>`,
+				answer: latestRecord.users[key] && latestRecord.users[key].answer  === 'fact'
+			};
+		});
```

Comment:
I think we need to sort the votes based on the timestamp they were submitted. One of the features we want is the votes are rendered in the order they were submitted.

Reply:
You are correct. :)